install/drain-cleaner/openshift/060-Deployment.yaml

```diff
@@ -15,17 +15,17 @@ spec:
       labels:
         app: strimzi-drain-cleaner
         com.company: Red_Hat
-        rht.prod_name: Red_Hat_Application_Foundations
-        rht.prod_ver: "2023.Q2"
-        rht.comp: AMQ_Streams
-        rht.comp_ver: "2.4"
-        rht.subcomp: drain-cleaner
-        rht.subcomp_t: application
+          rht.prod_name: Red_Hat_Application_Foundations
+          rht.prod_ver: "2023.Q3"
+          rht.comp: AMQ_Streams
+          rht.comp_ver: "2.5"
+          rht.subcomp: drain-cleaner
+          rht.subcomp_t: application
```

Comment:
wrong indentation?

Reply:
@ppatierno I was in the wrong yaml. Fixed now!!